src/program.ts

```diff
@@ -1833,8 +1836,30 @@ export class Program extends DiagnosticEmitter {
     );
     if (!parent.add(name, element)) return null;
     var memberDeclarations = declaration.members;
+    if (declaration.extendsType) queuedExtends.push(element);
+    var instanceDeclarations = new Array<DeclarationStatement>();
+    /**
+     * Must convert field declarations to property declarations
+     */
     for (let i = 0, k = memberDeclarations.length; i < k; ++i) {
       let memberDeclaration = memberDeclarations[i];
+      if (memberDeclaration.kind == NodeKind.FIELDDECLARATION) {
```

Comment:
Wondering why a second for loop is necessary here, and if this could instead be done in `initializeField` instead?